docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -0,0 +1,196 @@
+   
+3. Open a command prompt terminal window, and run the following command with the `.zip` file:
+   ```sh
+   CertUtil -hashfile <archive name>.zip SHA256
+   ```
+   Compare the returned value in the output with what's in the `<archive name>.zip.sha256` file:
+   * If the values are the same, you have downloaded the correct file successfully.
+   * If not, create a Support ticket [here](https://www.intel.com/content/www/us/en/support/contact-intel.html).
+
+4. Unzip the `<archive name>.zip` file using your preferred archive tool. 
+ 
+The standard OpenVINO `<INSTALL_DIR>` used in this document is `C:\Program Files (x86)\Intel\openvino_<version>\`. You're recommended to move the extracted files to that directory. 
+
+For simplicity, you can also create a symbolic link to the latest installation: `C:\Program Files (x86)\Intel\openvino_2022\`.
+
+To check **Release Notes** please visit: [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes).
+
+The core components are now installed. Continue to the next section to configure environment.
+
+### <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
+
+> **NOTE**: If you installed the Intel® Distribution of OpenVINO™ to a non-default install directory, replace `C:\Program Files (x86)\Intel` with that directory in this guide's instructions.
+
+You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables:
+
+```sh
+"<INSTALL_DIR>\setupvars.bat"
+```
+
+**Optional**: OpenVINO™ toolkit environment variables are removed when you close the command prompt window. You can permanently set the environment variables manually.
+
+> **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. Check your system environment variables, and add Python if necessary.
+
+The environment variables are set. Next, you can download some additional tools.
+
+### <a name="model-optimizer">Step 3 (Optional): Download Additional Components</a>
+
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. The OpenVINO™ Development Tools can only be installed via PyPI now. See [Install OpenVINO™ Development Tools](installing-model-dev-tools.md) for detailed steps.
+
+@sphinxdirective
+
+.. dropdown:: OpenCV
+
+   OpenCV is necessary to run demos from Open Model Zoo (OMZ). Some OpenVINO samples can also extend their capabilities when compiled with OpenCV as a dependency. The Intel® Distribution of OpenVINO™ provides a script to install OpenCV: ``<INSTALL_DIR>\extras\scripts\download_opencv.ps1``.
```

Comment:
This may change for 2022.2.

Reply:
For now, I think we can redirect users to the Github page. Updated.